packages/react-components/react-card/src/components/Card/useCard.ts

```diff
@@ -13,18 +13,28 @@ import { useFocusableGroup } from '@fluentui/react-tabster';
  * @param ref - reference to root HTMLElement of Card
  */
 export const useCard_unstable = (props: CardProps, ref: React.Ref<HTMLElement>): CardState => {
+  const { appearance = 'filled', focusMode = 'off' } = props;
+
+  const focusMap: Record<CardCommons['focusMode'], UseFocusableGroupOptions['tabBehavior']> = {
+    off: undefined,
+    'no-tab': 'limitedTrapFocus',
+    'tab-exit': 'limited',
+    'tab-only': 'unlimited',
+  };
+
   const groupperAttrs = useFocusableGroup({
-    tabBehavior: 'limitedTrapFocus',
+    tabBehavior: focusMap[focusMode],
   });
 
-  const { appearance = 'filled' } = props;
   return {
     appearance,
+    focusMode,
 
     components: { root: 'div' },
     root: getNativeElementProps(props.as || 'div', {
```

Comment:
**TIL:** this function is a ticking bomb:

![image](https://user-images.githubusercontent.com/1223799/166509184-c45d2859-b3ee-4d6e-935d-eab663cd9267.png)